src/main/java/hudson/plugins/copyartifact/TriggeredBuildSelector.java

```diff
@@ -201,7 +214,7 @@ public boolean isUseNewest() {
     
     @Override
     protected boolean isSelectable(Run<?,?> run, EnvVars env) {
-        return isFallbackToLastSuccessful() && run.getResult().isBetterOrEqualTo(Result.SUCCESS);
+        return isFallbackToLastSuccessful() && isBuildResultBetterOrEqualTo(run, Result.SUCCESS);
```

Comment:
```
    <BugInstance type='NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE'
        priority='Normal'
        category='STYLE'
        message='Possible null pointer dereference in hudson.plugins.copyartifact.TriggeredBuildSelector.isSelectable(Run, EnvVars) due to return value of called method'
        lineNumber='204'/>
```
Actually, `isSelectable` is expected to be called for only completed builds and `run.getResult()` should not be null.
(Though, wrapped with a helper method with null checks)